src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE_POSITION = 0;
+    private static final int NAME_POSITION = 1;
+    private static final int WORKING_HOURS_POSITION = 2;
+    private static final int INCOME_POSITION = 3;
+    private static final DateTimeFormatter DTF = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate from = LocalDate.parse(dateFrom, DTF);
+        LocalDate to = LocalDate.parse(dateTo, DTF);
```

Comment:
Consider adding error handling for date parsing to manage potential `DateTimeParseException` if the input date strings are not in the expected format. This will make your code more robust and user-friendly.